rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-criteria.js

```diff
@@ -70,9 +70,11 @@ export class SakaiRubricCriteria extends RubricsElement {
                   ${r.title}
                   <sakai-rubric-criterion-rating-edit criterion-id="${c.id}" @save-rating="${this.saveRating}" @delete-rating="${this.deleteRating}" minpoints="${c.pointrange ? c.pointrange.low : 0}" maxpoints="${c.pointrange ? c.pointrange.high : 0}" rating="${JSON.stringify(r)}"></sakai-rubric-criterion-rating-edit>
                 </h5>
-                <p>
+                <div class="div-description">
```

Comment:
Is this change inherited from #7298 ?

Reply:
Shouldn't be. I based off master. You have me worried though.

Reply:
Good job you spotted that. I must have copied some code from the deployed webapp. Nice one!